package/doc/sphinx/source/_templates/footer.html

```diff
@@ -0,0 +1,11 @@
+{% extends "!footer.html" %}
+
+{% block extrafooter %}
+{{ super() }}
+<div class="footer"><p>Please see
+    our <a href="https://www.mdanalysis.org/pages/privacy/">Privacy Policy</a>
+    to learn how <a href="https://www.mdanalysis.org">MDAnalysis</a> collects data.</p>
+    <script data-goatcounter="https://mdanalysis.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
+</div>
+{% endblock %}
```

Comment:
Given how many contributors there are, it might be more accessible to have the privacy policy before them?

```suggestion
{% block extrafooter %}
<div class="footer"><p>Please see
    our <a href="https://www.mdanalysis.org/pages/privacy/">Privacy Policy</a>
    to learn how <a href="https://www.mdanalysis.org">MDAnalysis</a> collects data.</p>
    <script data-goatcounter="https://mdanalysis.goatcounter.com/count"
        async src="//gc.zgo.at/count.js"></script>
</div>
{{ super() }}
{% endblock %}
```

Reply:
I find it easier to scroll to the bottom than midway, so I thought that the very bottom of a page is better real estate than somewhere higher up. But maybe I am biased because I mostly look at the docs on a screen and not on a small phone amd 

For the other docs (pmda, griddata etc) it's also at the very bottom (where I think most people would expect the privacy policy link) and it would get lost a little bit between the text block and the authors + sphinx/RTD links. 

By the way, the https://www.mdanalysis.org/GridDataFormats/ docs now render a bit differently (perhaps it uses Sphinx 3.1) and in this case I find at the very bottom also acceptable.

Reply:
Feel free to change if you think that switching order looks better.

Reply:
No, I see the logic in having it at the very bottom; I was concerned that people would stop scrolling before getting to the bottom but it's better to be consistent with the other docs.